apps/google-analytics-4/frontend/src/components/main-app/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+import tokens from '@contentful/f36-tokens';
+
+const ACCESSIBILITY_LABEL = 'Analytics line chart';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Title, Tooltip);
+
+ChartJS.defaults.font.size = 12;
+ChartJS.defaults.font.family = tokens.fontStackPrimary;
+ChartJS.defaults.font.weight = '500';
```

Comment:
avoided using a token here because of type conversion. Happy to convert a token of type **number** to a **string**,  but felt it was perhaps unnecessary for the benefit of using a token.

Reply:
i think we should consistently use the f36 tokens wherever possible. 12px font size should be `tokens.fontSizeS` and weight 500 should be `tokens.fontWeightMedium` – you can use base 16px to convert rems to px for the font size and typecast 500.toString() as you mentioned for the weight.

Reply:
Thats a fair point! I have adjusted this to better utilize the tokens 👍